cmd/tt/bucket.go

```diff
@@ -76,6 +78,62 @@ Existing configs will not be overwritten.
 			cmd.Fatal(fmt.Errorf("bucket %s is already initialized", root))
 		}
 
+		existing := configViper.GetBool("existing")
+
+		if existing {
+			ctx, cancel := authCtx(cmdTimeout)
+			defer cancel()
+
+			res, err := users.ListThreads(ctx)
+			if err != nil {
+				cmd.Fatal(err)
+			}
+
+			type bucketInfo struct {
+				ID   thread.ID
+				Name string
+				Key  string
+			}
+			var bucketInfos []bucketInfo
+			for _, reply := range res.List {
+				id, err := thread.Cast(reply.ID)
+				if err != nil {
+					cmd.Fatal(err)
+				}
+				ctx = common.NewThreadIDContext(ctx, id)
+				res, err := buckets.List(ctx)
+				if err != nil {
+					cmd.Fatal(err)
+				}
+				for _, root := range res.Roots {
+					name := "unnamed"
+					if root.Name != "" {
+						name = root.Name
+					}
+					bucketInfos = append(bucketInfos, bucketInfo{ID: id, Name: name, Key: root.Key})
+				}
+			}
+
+			prompt := promptui.Select{
+				Label: "Which exiting Bucket do you want to init from?",
+				Items: bucketInfos,
+				Templates: &promptui.SelectTemplates{
+					Active:   fmt.Sprintf(`{{ "%s" | cyan }} {{ .Name | bold }} {{ .Key | faint | bold }}`, promptui.IconSelect),
```

Comment:
in the case of an unnamed bucket, not sure which is best to display as a backup way to differentiate buckets... key or thread id?

Reply:
I'd say key as it's unique to the bucket